furnace.c

```diff
@@ -230,6 +258,37 @@ command_handler(furnace_context_t* ctx, uint8_t* buffer, void (*feedback)(const
     }
   }
 #endif
+#if CONFIG_WATER
+  else if(sscanf(buffer, "water %u", &arg) == 1) {
+    const int pwm_value = arg + WATER_OFFSET;
+    const int res = set_pwm_safe(WATER_PIN, ctx, pwm_value);
```

Comment:
First sanitize user input in range (0:10), then call set_pwm_safe. We should't be calling anything if the user input doesn't make sense.